.github/workflows/build_servers_cmake.yml

```diff
@@ -31,23 +31,19 @@ jobs:
       matrix:
           # The ubuntu-latest label currently points to ubuntu-22.04.
           # Available: ubuntu-22.04, ubuntu-20.04
-          os: [ubuntu-latest]
+          os: [ubuntu-latest, windows-latest]
```

Comment:
If cmake (ubuntu) is already used for the other builds, it might be wise to only run windows here.

Reply:
Renamed this to show it was just windows too